src/data/markdown/docs/02 javascript api/07 k6-experimental/04 tracing.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "tracing"
+excerpt: "k6 Tracing experimental API"
+---
+
+<ExperimentalBlockquote />
+
+
+This experimental module instruments HTTP requests in k6 scripts with tracing information.
```

Comment:
```suggestion
With this experimental module, you can _instrument_ you HTTP requests so that they emit traces as the test runs.
```

Reply:
It was quite a sage for me to understand what instrumentation means, and my first impression was that we should define it, but now I think we can settle to try to implicitly define it in the first sentence.

Does it look alright?

Reply:
This is perfect 👍🏻